src/merge_bed.py

```diff
@@ -29,12 +26,29 @@ def read_bed_file(f: TextIO) -> list[BedLine]:
     return res
 
 
-def merge(f1: list[BedLine], f2: list[BedLine], outfile: TextIO) -> None:
+def merge(f1, f2):
     """Merge features and write them to outfile."""
-    # FIXME: I have work to do here!
-
+    if f1 == '' or f1 == None:
+        f1 = []
+    if f2 == '' or f2 == None:
+        f2 = []
+    res = []
+    i,j = 0,0
+    B = 0
+    while B==0: 
+        if f1[i][0][-1] <= f2[j][0][-1] and f1[i][1] <= f2[j][1]:
+          res.append(f1[i])
+          i += 1
+        else:
+          res.append(f2[j])
+          j += 1
+        if i > len(f1)-1 or j > len(f2)-1:
+            B=1
+    res = res + f1[i:] + f2[j:]
+    return res
```

Comment:
Good, but why take the output file as an argument if you are not going to use it?